xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/lightbox.js

```diff
@@ -239,6 +240,47 @@ define('xwiki-lightbox', [
         $('.popover .copyImageId').parent().removeClass('hidden');
       }
     });
+    // We add the sr-only buttons to expand the lightbox
+    let toggler = $(JSON.parse($('#lightbox-config').text()).togglerTemplate);
+    lightboxImages.each(function() {
+      let lightboxToggle = toggler.clone();
+      lightboxToggle.get(0)
+        .addEventListener('focus', (e)=> {lightboxToggle.get(0).classList.remove('sr-only');});
+      lightboxToggle.get(0)
+        .addEventListener('focusout', (e) => {lightboxToggle.get(0).classList.add('sr-only');});
+      lightboxToggle.get(0)
+        .addEventListener('click', (e) => {
+        clearTimeout(showTimeout);
+        popoverContainer.data('target', this);
+        let offsetX = e.pageX;
+        let offsetY = e.pageY;
+        if (offsetX === 0 && offsetY === 0) {
+          // When the click is triggered from keyboard,
+          // we display the lightbox menu modal from the toggler.
+          let offset = lightboxToggle.offset();
+          offsetX = offset.left;
+          offsetY = offset.top;
+        }
+        popoverContainer.css({left: offsetX, top: offsetY});
+        popoverContainer.popover('show');
+        $('.openLightbox').focus();
+        // We add a tabbable element to the popover Container to avoid leaving the document focus
+        // when reaching the end of the popover with keyboard
+        popoverContainer.append($("<div id='popoverKeyboardEscaper' class='sr-only' tabindex='0'>"));
+        // When the popover is accessed through keyboard, it's also closed with keyboard navigation.
+        let focusLeavingPopover = function (event) {
+          event.preventDefault();
```

Comment:
The `focusout` event cannot be cancelled afaik.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2281/commits/ec73cce19023f83c8903af1f1f4d15133ef51762